scripts/test-lengths.py

```diff
@@ -0,0 +1,386 @@
+import sys
+import getopt
+from itertools import chain
+from random import sample
+
+from tlsfuzzer.runner import Runner
+from tlsfuzzer.messages import Connect, ClientHelloGenerator, \
+        ClientKeyExchangeGenerator, ChangeCipherSpecGenerator, \
+        FinishedGenerator, ApplicationDataGenerator, AlertGenerator, Close, \
+        SetMaxRecordSize
+from tlsfuzzer.expect import ExpectServerHello, ExpectCertificate, \
+        ExpectServerHelloDone, ExpectChangeCipherSpec, ExpectFinished, \
+        ExpectAlert, ExpectApplicationData, ExpectClose, \
+        ExpectServerKeyExchange
+
+from tlslite.constants import CipherSuite, AlertLevel, AlertDescription, \
+        GroupName, ExtensionType
+from tlslite.extensions import SupportedGroupsExtension, \
+        SignatureAlgorithmsExtension, SignatureAlgorithmsCertExtension, \
+        TLSExtension
+from tlsfuzzer.utils.lists import natural_sort_keys
+from tlsfuzzer.helpers import SIG_ALL, AutoEmptyExtension
+
+
+version = 2
+
+
+def help_msg():
+    print("Usage: <script-name> [-h hostname] [-p port] [[probe-name] ...]")
+    print(" -h hostname    name of the host to run the test against")
+    print("                localhost by default")
+    print(" -p port        port number to use for connection, 4433 by default")
+    print(" probe-name     if present, will run only the probes with given")
+    print("                names and not all of them, e.g \"sanity\"")
+    print(" -e probe-name  exclude the probe from the list of the ones run")
+    print("                may be specified multiple times")
+    print(" -x probe-name  expect the probe to fail. When such probe passes despite being marked like this")
+    print("                it will be reported in the test summary and the whole script will fail.")
+    print("                May be specified multiple times.")
+    print(" -X message     expect the `message` substring in exception raised during")
+    print("                execution of preceding expected failure probe")
+    print("                usage: [-x probe-name] [-X exception], order is compulsory!")
+    print(" -n num         run 'num' or all(if 0) tests instead of default(all)")
+    print("                (\"sanity\" tests are always executed)")
+    print(" -d             use TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256")
+    print("                instead of TLS_RSA_WITH_AES_128_CBC_SHA256")
+    print(" -C cipher      ciphersuite to use for the connection")
+    print(" --extra-exts   send also supported_groups, signature_algorithms,")
+    print("                and signature_algorithms_cert extensions in Client")
+    print("                Hello. Default for DHE and ECDHE ciphers")
+    print(" -t timeout     Connection and server reply timeout, in seconds.")
+    print("                5 seconds by default.")
+    print(" --etm          Advertise and expect encrypt_then_mac extension")
+    print(" --size-limit num Send a max_fragment_length extension advertising")
```

Comment:
Please check this line.

Reply:
I don't think there's an issue here, leaving as-is

Reply:
Pardon, I misunderstood this line and there is actually no issue.